spec.html

```diff
@@ -26731,13 +26751,13 @@ <h1>Cyclic Module Records</h1>
             </tr>
             <tr>
               <td>
-                [[AsyncEvaluation]]
+                [[AsyncEvaluationOrder]]
               </td>
               <td>
-                a Boolean
+                ~unset~, an integer, or ~done~
```

Comment:
```suggestion
                an integer, ~unset~, or ~done~
```
(editorial convention says to put types before constants)

Reply:
I actually did this on purpose, since the value "evolves" from unset, to an integer, to done.

Reply:
I'm okay either way. The editors can decide whether to enforce the convention or allow the variance.

Reply:
I like the way it's written in the PR.